### DISCUSSION TITLE
Version Controlling

### DISCUSSION THREAD

**Discussion by bliu13** [SpiralLinux/SpiralLinux-project#81] [Q&A] [answered]
Hi, I wanted to ask a stupid question. Was there a specific reason for compressing the build scripts in the repository? I think git does not diff binary blobs so when it takes a snapshot, it would treat each binary as a new blob to save. If you make 10 changes to a 10 MB tar.gz file, wouldn't that cause the repo to have a size of 100 MB? If only some files are going to change, wouldn't it be better if the tar.gz files were not compressed?

**Comment by geckolinux** [comment, marked as answer]
> Was there a specific reason for compressing the build scripts in the repository?

Yep, the reason is embarrassingly simple. ;-) I barely know how to use git, and I'm not really interested in using the features of git because it's overkill for my simple workflow. So it's easier for me to to just create tarballs of the build directories and drag-n-drop them to the Github web interface.
